README.md

```
@@ -119,10 +122,12 @@ Please keep in mind:
 More tips:
 - Guides to setup your development environment for [IntelliJ IDEA](https://github.com/dbpedia/extraction-framework/wiki/Setting-up-intellij-idea) or [Eclipse](https://github.com/dbpedia/extraction-framework/wiki/Setting-up-eclipse).
-- Get help with the [Maven build](Build-from-Source-with-Maven) or another form of [installation](Installation).
-- [Download](Downloads) some data to work with.
-- How to run [from Scala/Java](Run-from-Java-or-Scala) or [from a JAR](Run-from-a-JAR).
-- Having different troubles? Check the [troubleshooting page](Troubleshooting) or post on https://forum.dbpedia.org.
+- Get help with the Maven build or another form of installation.
+<!-- Broken links removed: previously linked to Build-from-Source-with-Maven and Installation pages -->
+- Download some data from [DBpedia Downloads](https://www.dbpedia.org/resources/downloads/) to work with.
+- How to run from Scala/Java or from a JAR.
+<!-- Broken links removed: previously linked to Run-from-Java-or-Scala and Run-from-a-JAR pages -->
+- Having different troubles? Check the project wiki or post on https://forum.dbpedia.org.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)